### ISSUE TITLE
[Enhancement] Colored tags would be nice.

### ISSUE THREAD

**Issue by eToThePi-i** [firasdib/Regex101#987]
Colored tags would allow easier regex assessment at a glance, at least in my organizational system.

**Comment by firasdib** [comment]
Yes, that would be nice, but question is how it could fit in with the UI in a reasonable way.

Currently, tags are added and modified in a modal - when and where should the tag coloring be changed? Should all tags be unique, or should those with the same value be considered same, regardless of color? There are a few outstanding questions to make this work well.

**Comment by soleous** [comment]
I could really need this.  I am dyslexic and I cannot read the code because of the colours of the tags, so I cannot use the site.  Is this planned for? or drop dead.
